### PR TITLE
Ensure LAMMPS gets sent integer time values. 

### PR DESCRIPTION
**Pull request by clpetix**
Addresses Issue #260

### CODE REVIEW

File: src/relentless/simulate/lammps.py [hunk 1]
```diff
@@ -1641,3 +1643,10 @@ def _initialize_engine(self, sim):
     _EnsembleAverage = EnsembleAverage
     _Record = Record
     _WriteTrajectory = WriteTrajectory
+
+
+def to_int(a):
```
Comment (mphoward) [inline src/relentless/simulate/lammps.py:1648]: Let's make this explicitly private. These modules are really part of the public API, but I don't want to promise this function will always work. You should also add a doc string.
```suggestion
def _to_int(a):
```

File: tests/simulate/test_lammps.py [hunk 2]
```diff
@@ -246,6 +246,28 @@ def test_minimization(self):
         lmp.operations = emin
         lmp.run(potentials=pot, directory=self.directory)
 
+        # Whole number of iterations and evalulations as float
+        emin.max_iterations = 1000.0
```
Comment (mphoward) [inline tests/simulate/test_lammps.py:250]: Did you confirm these tests fail without the bugfix?

File: tests/simulate/test_lammps.py [hunk 3]
```diff
@@ -246,6 +246,28 @@ def test_minimization(self):
...
+        emin.options = {"max_evaluations": 10000.0}
+        lmp.operations = emin
+        lmp.run(potentials=pot, directory=self.directory)
+        emin.max_iterations = 1000
+        emin.options = {"max_evaluations": 10000}
```
Comment (mphoward) [inline tests/simulate/test_lammps.py:255]: What is the purpose of these lines here? It wasn't clear to me.

File: tests/simulate/test_lammps.py [hunk 4]
```diff
@@ -258,6 +280,18 @@ def test_brownian_dynamics(self):
         else:
             h.run(pot, self.directory)
 
+        # Whole number of steps as float
+        brn.steps = 1.0
+        h = relentless.simulate.LAMMPS(init, brn, executable=self.executable)
```
Comment (mphoward) [inline tests/simulate/test_lammps.py:285]: You don't need to make new simulation engines to do a new run, remove these extra instances throughout.

**Review by mphoward — CHANGES_REQUESTED** [review]
Thanks! Just a few small things and then this can be merged.

File: tests/simulate/test_lammps.py (hunk 2 above)
Comment (clpetix) [reply tests/simulate/test_lammps.py:250]: yes

File: tests/simulate/test_lammps.py (hunk 3 above)
Comment (clpetix) [reply tests/simulate/test_lammps.py:255]: The order of those tests are: 
1.) whole number iterations and max evaluations as floats
2.) Non whole number of iterations
3.) Non whole number of max evaluations

I thought it was best to reset it after 1 to the default integer state before testing 2 & 3.

Comment (mphoward) [reply tests/simulate/test_lammps.py:255]: OK! Maybe add some comments documenting that the first time you use that pattern?

Comment (clpetix) [reply tests/simulate/test_lammps.py:257]: I added comments whenever I reset something. I think that should help the readability!

**Review by mphoward — APPROVED** [review]
Thank you! This is a clean solution.

**Pull request merged**